jeremywiebe/harp
# Update from upstream

Importing the latest updates from upstream harp.

Most of these updates are just package.json changes which help to eliminate many of the deprecation warnings.
  
## Testing
- Go to the progressive-web-sdk directory
- Update `package.json` so that the harp dependency points to this branch.
   ```json
   ...
   "dependencies": {
      ...
      "harp": "mobify/harp#update-from-upstream",
      ...
   }
   ...
   ```
- `npm install`
- `npm run docs:dev`
- Ensure the docs generate and work
- `npm run docs:compile`
- Run `docs/preview-compiled-docs.sh`
